testing/py-scikit-learn/APKBUILD

```diff
@@ -0,0 +1,58 @@
+# Contributor: Carlos Giraldo <cgiraldo@gradiant.org>
+# Maintainer: Carlos Giraldo <cgiraldo@gradiant.org>
+pkgname=py-scikit-learn
+_pkgname=scikit-learn
+pkgver=0.20.3
+pkgrel=0
+pkgdesc="scikit-learn is a Python module for machine learning built on top of SciPy"
+url="http://scikit-learn.org/"
+arch="all"
+license="BSD"
```

Comment:
The license should be a valid [SPDX identifier][SPDX], which just `BSD` is not. Please clarify which actual license it is.

[SPDX]:https://spdx.org/licenses/

Reply:
Changed to BSD-3-Clause